pkg/kubelet/kubelet.go

```diff
@@ -2366,6 +2371,9 @@ func (kl *Kubelet) setNodeStatus(node *api.Node) error {
 		if !networkConfigured {
 			messages = append(reasons, "network not configured correctly")
 		}
+		if !selinuxConfigured {
+			messages = append(reasons, "SELinux is required by the cluster but not enabled on this node")
```

Comment:
@dchen1107 Heterogenous is not allowed, correct.  I don't know a single person / group that wants it.